src/stack-hci/azext_stack_hci/generated/commands.py

```diff
@@ -44,21 +44,28 @@ def load_command_table(self, _):
         g.custom_command('list', 'stack_hci_arc_setting_list')
         g.custom_show_command('show', 'stack_hci_arc_setting_show')
         g.custom_command('create', 'stack_hci_arc_setting_create')
+        g.custom_command('update', 'stack_hci_arc_setting_update')
         g.custom_command('delete', 'stack_hci_arc_setting_delete', supports_no_wait=True, confirmation=True)
+        g.custom_command('create-identity', 'stack_hci_arc_setting_create_identity', supports_no_wait=True)
+        g.custom_command('generate-password', 'stack_hci_arc_setting_generate_password')
         g.custom_wait_command('wait', 'stack_hci_arc_setting_show')
 
     with self.command_group('stack-hci cluster', stack_hci_cluster, client_factory=cf_cluster) as g:
         g.custom_command('list', 'stack_hci_cluster_list')
         g.custom_show_command('show', 'stack_hci_cluster_show')
         g.custom_command('create', 'stack_hci_cluster_create')
         g.custom_command('update', 'stack_hci_cluster_update')
-        g.custom_command('delete', 'stack_hci_cluster_delete', confirmation=True)
+        g.custom_command('delete', 'stack_hci_cluster_delete', supports_no_wait=True, confirmation=True)
+        g.custom_command('create-identity', 'stack_hci_cluster_create_identity', supports_no_wait=True)
+        # service team found a bug on this api: Operation returned an invalid status 'OK'
+        # g.custom_command('upload-certificate', 'stack_hci_cluster_upload_certificate', supports_no_wait=True)
```

Comment:
May I ask has this problem been communicated with the server team? Do they agree not to publish this command first?

Reply:
https://github.com/Azure/azure-cli/issues/22470#issuecomment-1193698919